eng/scripts/DownloadDevPackages.ps1

```diff
@@ -0,0 +1,50 @@
+param (
+    [Parameter(Mandatory=$True)]
+    [string] $WorkingDirectory,
+    [Parameter(Mandatory=$True)]
+    [string] $NupkgFilesDestination,
+    [string] $NugetSource="https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json"
+)
+
+. (Join-Path $PSScriptRoot ".." common scripts common.ps1)
+
+$allPackages = Get-AllPkgProperties
+$trackTwoPackages = $allPackages.Where({ $_.IsNewSdk })
+
+Push-Location $WorkingDirectory
+$nugetPackagesPath = Join-Path $WorkingDirectory nugetPackages
+New-Item -Path $WorkingDirectory -Type "directory" -Name "nugetPackages"
+
+$packagesInFeed = Find-Package -Source $NugetSource -AllVersion -AllowPrereleaseVersions
```

Comment:
We should see how slow this is once we are done as it might end up taking a while.

Reply:
This was slow and it did not return all the packages. I sitche to calling the DevOps Feed api.